modules/services/lorri.nix

```
@@ -6,18 +6,48 @@ let
 
   cfg = config.services.lorri;
 
+  jqFile = ''
+    (
+      (.Started?   | values | "Build starting in \(.nix_file)"),
+      (.Completed? | values | "Build complete in \(.nix_file)"),
+      (.Failure?   | values | "Build failed in \(.nix_file)")
+    )
+  '';
+
+  notifyScript = pkgs.writeTextFile {
+    name = "lorri_notify";
+    executable = true;
+    text = ''
+      #! /usr/bin/env bash
+
+      lorri internal stream-events --kind live |\
+      jq --unbuffered '${jqFile}' |\
+      xargs -n 1 notify-send "Lorri Build"
+    '';
+  };
```

Can simplify using `writeShellScript`:
```suggestion
  notifyScript = pkgs.writeShellScript "lorri-notify" ''
    lorri internal stream-events --kind live \
      | jq --unbuffered '${jqFile}' \
      | xargs -n 1 notify-send "Lorri Build"
  '';
```

---

I guess the thing here is that I'm not sure how useful the script is outside the specific context of a per-user systemd service. There's some discussion in the lorri issues about whether it makes sense to standardize this as a lorri subcommand - I'm not sure there's a universal version that makes sense.